partcad/src/partcad/runtime_python.py

```
@@ -16,10 +16,47 @@
 
 from . import runtime
 from . import logging as pc_logging
+from . import sync_threads
+
+
+class VenvLock(object):
+    def __init__(self, runtime, venv: str):
+        runtime.venv_locks_lock.acquire()
+        if not venv in runtime.venv_locks:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)